server/app/modules/join_hunt/router_models.py

```diff
@@ -0,0 +1,21 @@
+from typing import List, Optional, Union
+from datetime import datetime
+from pydantic import BaseModel, Field
+from app.models import PyObjectId
+from app.modules.users.router_models import UserModel
+
+class Join_Hunts_Model(BaseModel):
```

Comment:
The fields in this do not match the usage in router.py

Reply:
Change this so that the fields are the same: message, and content, as you seem to have put above in the API response. Make sure you implement the right fields for success and failure cases (should it just be the "message" field? so on)